__tests__/addSerializers.test.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+const React = require('react');
+const { configure, shallow } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+const enzymeToJson = require('enzyme-to-json/serializer');
+const snapshotDiff = require('../src/index');
+
+configure({ adapter: new Adapter() });
+snapshotDiff.addSerializers([enzymeToJson]);
+
+type Props = {
+  test: string,
+};
+
+class Component extends React.Component<Props> {
+  render() {
+    return (
+      <div>
+        <span />
+        <span />
+        <span />
+        <span>{this.props.test}</span>
+        <span>{this.props.test}</span>
```

Comment:
Both shallow and full render will look the same for this component, so this test doesn't really shows if this serializer works. I think instead of adding an example with massive `span`s amount, it would be better to create a `NestedComponent` nested inside these divs or spans, and see if they render shallowly. Or make it as another test case, to see if multiple serializers can live with each other

Reply:
Could we also add a small test with React's shallow renderer? https://reactjs.org/docs/shallow-renderer.html